components/script/script_thread.rs

```diff
@@ -1173,25 +1180,17 @@ impl ScriptThread {
     ///
     /// Attempt to update the rendering and then do a microtask checkpoint if rendering was actually
     /// updated.
-    pub(crate) fn update_the_rendering(&self, requested_by_renderer: bool, can_gc: CanGc) {
+    pub(crate) fn update_the_rendering(&self, can_gc: CanGc) {
         *self.last_render_opportunity_time.borrow_mut() = Some(Instant::now());
 
         // If the ScriptThread animation timer fired, this is an animation tick.
-        let mut is_animation_tick = requested_by_renderer;
-        if self
+        let is_animation_tick = self
             .should_trigger_script_thread_animation_tick
-            .load(Ordering::Relaxed)
-        {
+            .load(Ordering::Relaxed);
+        if is_animation_tick {
             self.should_trigger_script_thread_animation_tick
                 .store(false, Ordering::Relaxed);
-            *self.scheduled_script_thread_animation_timer.borrow_mut() = None;
```

Comment:
Why is this line of code removed?

Reply:
because later on we do `take` on option, that we use to cancel queued event